action.yml

```diff
@@ -33,5 +36,5 @@ runs:
         # the request metadata to pkg.julialang.org when installing
         # packages via `Pkg.test`.
         JULIA_PKG_SERVER: ""
-    - run: julia --color=yes --check-bounds=yes --inline=${{ inputs.inline }} --project -e 'using Pkg; Pkg.test(coverage=${{ inputs.coverage }})'
+    - run: julia --color=yes --check-bounds=yes --inline=${{ inputs.inline }} --project -e 'using Pkg; kwargs = Dict{Symbol, Any}(); kwargs[:coverage] = ${{ inputs.coverage }; ${{ inputs.force_latest_compat }} && (kwargs[:force_latest_compat] = ${{ inputs.force_latest_compat }}); Pkg.test(; kwargs...)'
```

Comment:
This should probably be put on multiple lines

Reply:
Sure! I'm not sure what the syntax for that is.

Reply:
@christopher-dG Is this correct?
```yaml
- run: |
  julia --color=yes --check-bounds=yes --inline=${{ inputs.inline }} --project -e '
using Pkg;
kwargs = Dict{Symbol, Any}();
kwargs[:coverage] = ${{ inputs.coverage }; ${{ inputs.force_latest_compat }} && (kwargs[:force_latest_compat] = ${{ inputs.force_latest_compat }});
Pkg.test(; kwargs...)'
  shell: bash
```

Reply:
That looks good. I think you don't even need the semicolons.

Reply:
Except do indent the string by two more spaces.

Reply:
Alright, how about this:
```yaml
- run: |
  julia --color=yes --check-bounds=yes --inline=${{ inputs.inline }} --project -e '
    using Pkg
    kwargs = Dict{Symbol, Any}()
    kwargs[:coverage] = ${{ inputs.coverage }
    ${{ inputs.force_latest_compat }} && (kwargs[:force_latest_compat] = ${{ inputs.force_latest_compat }})
    Pkg.test(; kwargs...)'
  shell: bash
```

Reply:
The whole thing needs to be indented past the parent `run`, so:

```yml
- run: |
    julia ... -e '
      using Pkg ...'
```

Reply:
Like this?

```yaml
- run: |
    julia --color=yes --check-bounds=yes --inline=${{ inputs.inline }} --project -e '
      using Pkg
      kwargs = Dict{Symbol, Any}()
      kwargs[:coverage] = ${{ inputs.coverage }
      ${{ inputs.force_latest_compat }} && (kwargs[:force_latest_compat] = ${{ inputs.force_latest_compat }})
      Pkg.test(; kwargs...)'
  shell: bash
```

Reply:
Yep that parses. This seems like the dumbest nitpick ever but YAML really is this picky 😅

Reply:
Third time's the charm 😂 

Yeah this YAML stuff always kills me.

I've updated this PR - can you double check the syntax one last time?

Also, I've updated https://github.com/julia-actions/composite-runtest/pull/1 with the same change - can you check the syntax there as well?

Reply:
Both look good to me.